docs/guides/vpn/wireguard/internal.md

```diff
@@ -26,7 +26,25 @@ net.ipv6.conf.all.forwarding = 1
 
 they were enabled successfully.
 
-A properly configured firewall is ***highly*** recommended for any Internet-facing device. Configuring a firewall (`iptables`, `ufw`, etc.) is not part of this guide.
+A properly configured firewall is ***highly*** recommended for any Internet-facing device. A complete configuration of a firewall (`iptables`, `ufw`, etc.) is not part of this guide.
+
+## Open and close a route from the Wireguard server to the client
+
+<!-- markdownlint-disable code-block-style -->
+!!! warning "**Important:** Substitute interface"
+    **Without the correct interface name, this will not work!**
+
+    Substitute `eth0` in the commands below and commands to enable the NAT to match the Internet-facing interface. This may be `enp2s0` or similar on more recent Ubuntu versions. If you are unsure, you can use `ip a` to find the correct interface name. The interface name is the one that is connected to the Internet.
```

Comment:
```suggestion
    Substitute `eth0` in the commands below to enable the NAT to match the Internet-facing interface. This may be `enp2s0` or similar on more recent Ubuntu versions. If you are unsure, you can use `ip a` to find the correct interface name. The interface name is the one that is connected to the Internet.
```